scribble/app/views/posts/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>Scribble Posts</h1>
+<%= link_to "New post", new_post_path%>
+
+<table>
```

Comment:
It's best never to use a table for layout. These should be semantic tags like `<article>` or `<section>` or at least `<div>`s.

Tables should only be used for tabular data.